galvan-support/src/main/java/org/terracotta/testing/rules/BasicExternalCluster.java

```diff
@@ -108,7 +108,19 @@ public BasicExternalCluster(File clusterDirectory, int stripeSize, List<File> se
 
   @Override
   public Statement apply(Statement base, Description description) {
```

Comment:
Did u test this?  I was pretty sure the issue was actually at line BasicExternalCluster:111 when it calls
File testParentDirectory = File.createTempFile(this.displayName, "", this.clusterDirectory);
This is the code that creates a directory name with the name of the fully qualified name followed by a random number.